transaction/_transaction.py

```diff
@@ -22,6 +23,7 @@
 from transaction.weakset import WeakSet
 from transaction.interfaces import TransactionFailedError
 from transaction import interfaces
+from transaction._compat import binary_type
```

Comment:
Sure. I was just being consistent with the rest of the package.

I'll change the rest of the package to use ``bytes``.